src/domains/wallet/pages/CreateWallet/CreateWallet.tsx

```diff
@@ -214,29 +259,22 @@ export const CreateWallet = ({ networks, mnemonic, onSubmit, onCopy, onDownload,
 		setActiveTab(activeTab + 1);
 	};
 
-	const crumbs = [
-		{
-			route: `/profiles/${activeProfile?.id()}/dashboard`,
-			label: "Go back to Portfolio",
-		},
-	];
-
 	return (
 		<Page crumbs={crumbs}>
 			<Section className="flex-1">
-				<Form className="max-w-xl mx-auto" context={form} onSubmit={(data: any) => onSubmit(data)}>
+				<Form className="max-w-xl mx-auto" context={form} onSubmit={(data: any) => submitForm(data)}>
```

Comment:
Change the `onSubmit` to `onSubmit={submitForm}>`

Reply:
done :bleh: